src/ucp/core/ucp_worker.c

```diff
@@ -2045,39 +2045,6 @@ static void ucp_worker_vfs_show_address_name(void *obj,
     UCS_ASYNC_UNBLOCK(&worker->async);
 }
 
-static void ucp_worker_vfs_show_num_all_eps(void *obj,
-                                            ucs_string_buffer_t *strb,
-                                            void *arg_ptr, uint64_t arg_u64)
-{
-    ucp_worker_h worker = obj;
-
-    UCS_ASYNC_BLOCK(&worker->async);
```

Comment:
maybe add ucp_worker_vfs_show_primitive(), which will acquire the lock and then call ucs_vfs_show_primitive?

Reply:
better to have a common lock for all getters